### PR TITLE
rename message_type_support{_struct}.h

### PR DESCRIPTION
**Pull request by wjwwood**
Connects to ros2/rosidl#71

### CODE REVIEW

**Comment by jacquelinekay** [comment]
+1

**Comment by jacquelinekay** [comment]
I retract my +1, I think I understand what needs to happen now

File: rosidl_typesupport_opensplice_c/package.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <buildtool_depend>rosidl_generator_c</buildtool_depend>
+  <buildtool_depend>rosidl_typesupport_opensplice_cpp</buildtool_depend>
+
+  <buildtool_export_depend>ament_cmake</buildtool_export_depend>
+  <buildtool_export_depend>libopensplice64</buildtool_export_depend>
+  <buildtool_export_depend>opensplice_cmake_module</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_generator_c</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_generator_dds_idl</buildtool_export_depend>
+  <!-- The following is needed because the OpenSplice IDL generator is in that package. -->
```
Comment (jacquelinekay) [inline rosidl_typesupport_opensplice_c/package.xml]: Will we eventually move the IDL generator into a separate package that both Opensplice typesupport packages depend on?

**Comment by jacquelinekay** [comment]
the non-isolated build wasn't working because `ament_package` wasn't being called in CMakeLists.txt. fixed in 12fc1f9

+1

**Comment by wjwwood** [comment]
Thanks for the fixup @jacquelinekay, that was another example of me trying to propose part of my changes and failing to include everything that was needed.

**Pull request merged**